src/components/subscriptions/RemindUserButton.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LicenseRemindModal from '../../containers/LicenseRemindModal';
+import ActionButtonWithModal from '../ActionButtonWithModal';
+
+const RemindUserButton = ({
+  onSuccess,
+  onClose,
+  pendingUsersCount,
+  isBulkRemind,
+}) => (
+  <ActionButtonWithModal
+    buttonLabel="Remind all"
+    buttonClassName="btn btn-primary float-right"
+    renderModal={({ closeModal }) => (
+      <LicenseRemindModal
+        pendingUsersCount={pendingUsersCount}
+        isBulkRemind={isBulkRemind}
+        title="Remind Users"
+        onSuccess={onSuccess}
+        onClose={() => {
```

Comment:
indentation needs to be fixed
```
  <ActionButtonWithModal
    buttonLabel="Remind all"
    buttonClassName="btn btn-primary float-right"
    renderModal={({ closeModal }) => (
      <LicenseRemindModal
        pendingUsersCount={pendingUsersCount}
        isBulkRemind={isBulkRemind}
        title="Remind Users"
        onSuccess={onSuccess}
        onClose={() => {
          closeModal();
          if (onClose) {
            onClose();
          }
        }}
      />
    )}
  />

```
Also, be sure to check indentation in other places of the code.